samples/transform-string-stream/src/index.ts

```diff
@@ -7,6 +7,6 @@ const app: TransformApp = (
     suffix: string = "|Suffix"
 ) =>
     DataStream.from(input)
-        .map(name => prefix + name + suffix);
+        .map(name => prefix + name + suffix + "\n");
```

Comment:
question from curiosity: was it changed from cosmetic reasons or the Sequence didn't work as expected?

Reply:
It worked just fine, this is just a cosmetic correction :)

Reply:
cool :)